scripts/jenkins/ardana/ansible/roles/ardana_update/tasks/reboot_nodes.yml

```diff
@@ -32,19 +32,7 @@
   args:
     chdir: "{{ ardana_scratch_path }}"
 
-- name: Get list of remaining nodes to reboot
-  shell: |
-    deployer=`grep -A 1 ARD-SVC--first-member hosts/verb_hosts | tail -1`
-    for line in $(awk '/resources/' RS= hosts/verb_hosts); do
-      if [[ $deployer != *$line* ]]; then
-        echo $line
-      fi
-    done
-  args:
-    chdir: "{{ ardana_scratch_path }}"
-  register: ardana_nodes
-
-- name: Reboot remaining nodes serially
+- name: Reboot non-deployer nodes serially
```

Comment:
This would be painful for a scale install, ideally we want to do the control plane serially to avoid the galera issues, but do the compute nodes in parallel (I think the default parallelism is 20).

Reply:
@mnmehta , I agree, but then again, these scripts weren't meant to be used with a scale install. Are you saying you're using them to install updates on the Ardana scale cloud (or planning to) ? If that's the case, we need to do some more work to make them capable of that.

Reply:
@stefannica I am using the automation scripts for the scale install. However, right now I use the automation to install the control plane and 1 compute node, I add the rest of the compute nodes later in batches using different scripting. It would be nice to use the automation to install the entire cluster. Perhaps just add a backlog item to revisit this serial reboot (at least for all nodes which do not host galera)?

Reply:
> @stefannica I am using the automation scripts for the scale install. However, right now I use the automation to install the control plane and 1 compute node, I add the rest of the compute nodes later in batches using different scripting. It would be nice to use the automation to install the entire cluster. Perhaps just add a backlog item to revisit this serial reboot (at least for all nodes which do not host galera)?

@mnmehta that I can definitely do :) . Feel free to add your suggestions [here](https://jira.suse.de/browse/SCRD-8727)

Reply:
It should be possible to construct, using ansible inventory host groups, a set of control plane nodes, and a set of compute nodes, and run the required plays against each independently with --limit, serialising the control plane ones, and parallelising the computes...

The --limit option takes ansible host specs as argument, including the various operators to get the intersection, exclusion etc between host groups...